streamer/src/sendmmsg.rs

```diff
@@ -143,22 +122,13 @@ where
     S: Borrow<SocketAddr>,
     T: AsRef<[u8]>,
 {
-    // The vectors are allocated with capacity, as later code inserts elements
-    // at specific indices, and uses the address of the vector index in hdrs
-    let mut iovs: Vec<iovec> = Vec::with_capacity(packets.len());
-    let mut addrs: Vec<sockaddr_storage> = Vec::with_capacity(packets.len());
-    let mut hdrs: Vec<mmsghdr> = Vec::with_capacity(packets.len());
-
-    for (pkt, dest) in packets.iter() {
-        mmsghdr_for_packet(
-            pkt.as_ref(),
-            dest.borrow(),
-            &mut iovs,
-            &mut addrs,
-            &mut hdrs,
-        );
+    let size = packets.len();
+    let mut iovs = vec![unsafe { std::mem::zeroed() }; size];
```

Comment:
iovs could use mem::MaybeUninit::uninit().assume_init() as these will always be completely specified

Reply:
done